tests/integration/test_models/test_full_battery_models/test_lithium_ion/test_spm_half_cell.py

```diff
@@ -0,0 +1,75 @@
+#
+# Tests for the lithium-ion SPM model
+#
+import pybamm
+import tests
+
+import unittest
+
+
+class TestSPM(unittest.TestCase):
+    def test_basic_processing(self):
+        options = {"working electrode": "positive"}
+        model = pybamm.lithium_ion.SPM(options)
+        param = pybamm.ParameterValues(chemistry=pybamm.parameter_sets.Xu2019)
+        modeltest = tests.StandardModelTest(model, parameter_values=param)
+        modeltest.test_all(skip_output_tests=True)
+
+
+class TestDFNWithSEI(unittest.TestCase):
+    def test_well_posed_constant(self):
```

Comment:
Does SEI still work if the working electrode is the positive one?

Reply:
The SEI grows on the surface of the li metal electrode